routes/harvest-logs.routes.ts

```diff
@@ -1,12 +1,12 @@
-import express from "express";
 import HarvestLogSchema from "project-2-types/dist/ajv/harvest-log.ajv";
+import express from "express";
 import harvestController from "../controllers/harvest-log.controllers";
-import paths from "../shared/paths";
 import harvestLogValidator from "../shared/validators";
+import paths from "../shared/paths";
 
 const harvestLogRouter = express.Router();
 
-// POST: /api/v1/harvest-logs
+// POST: /api/v1/harvest-logs:id
 harvestLogRouter.post(
   paths.harvestLog,
```

Comment:
you have to add the :id to the path

<img width="756" alt="Screenshot 2024-03-08 at 3 36 38 PM" src="https://github.com/LangaraVoyagers/project-2-api/assets/57273227/86b3f1e4-64dc-43fc-a66a-c89390f9db8f">

Reply:
`${paths.harvestLog}/:id`

Reply:
This might have been caused during merge... I has it there... Will make the change